server/models/availability.js

```diff
@@ -0,0 +1,49 @@
+const mongoose = require("mongoose");
+
+
+const dayInformation={
+    active:{
+        type:Boolean,
+        default:'true'
+    },
+    startTime:{
+        type:String,
+        required:this.active
+    },
+    endTime:{
+        type:String,
+        required:this.active,
+    }
+}
+
+const availabilitySchema = new mongoose.Schema({
+  petSitterId: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+    ref: 'Profile'
+  },  
+  name: {
+    type:'string',
+    required:true
+  },
+  days: {
+  sunday:dayInformation,
+  monday:dayInformation,
+  tuesday:dayInformation,
+  wednesday:dayInformation,
+  thursday:dayInformation,
+  friday:dayInformation,
+  saturday:dayInformation
+  },
+  
+});
+availabilitySchema.pre('validate', function(next) {
+    var startHour = this.startTime.split(":")[0];
+    var endHour=this.endTime.split(":")[0];
+    if (startHour > endHour) {
```

Comment:
Additionally, we would want to check whether the hours/minutes are valid and make sense for the specified time (for example, 25:85 can still be a valid time since we are using strings)

Reply:
added the time validation